src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Endpoint.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, Response>
+{
+    private readonly AquiferDbContext _dbContext = dbContext;
```

Comment:
This is a deviation from what looked like standard practice.

I'm fairly new to primary constructors and I don't like the following:
1. The variable is in-scope for my whole class but looks just like a local variable in its naming (`dbContext in this case).
2. I can't mark the variable as read-only.

Therefore, I kept the primary constructor syntax but assigned it to a `private readonly` field here.  The compiler will nicely yell at you if you try to now use `dbContext` elsewhere in the class (solving my first concern) and the field begins with `_` which helps me to recognize it as a field instead of a local variable (solving my second concern).

Reply:
At that point do we gain benefit from having the primary constructor? Is .NET smart enough to dependency inject if you just had: `private readonly AquiferDbContext _dbContext` declared?

Reply:
This is just a reality of the primary constructor (I saw people complaining about this while the feature was in preview). Having it declared a second time here is redundant and makes no sense to me. To me the injected services are consistent and it should generally be obvious that's what they are when you call them in the code. There's always a tradeoff with syntactic sugar, and in this case I wouldn't want to go cluttering up classes with constructors and props over a naming issue or because we're worried that someone's gonna try and set an injected value (which shouldn't pass PR review any more than if they did it after injecting it and _not_ setting it as `readonly`).

Reply:
> At that point do we gain benefit from having the primary constructor?

Not really.  This is why I haven't used primary constructors much and have opted to keep using standard constructors with boilerplate code to assign fields.

> This is just a reality of the primary constructor.

Yep.  I was following the [Microsoft advice](https://devblogs.microsoft.com/dotnet/csharp-primary-constructors-refactoring/#additional-considerations-🤔) for keeping `readonly` which I really like being enforced by the compiler instead of convention or code review.

Thanks for hearing me out.  I'll switch it back to just use the standard primary constructor syntax but wanted to get your thoughts on it first.

Reply:
Removed [here](https://github.com/BiblioNexusStudio/aquifer-server/pull/443/commits/3c7bb04481abd2addde6afb8c36e8d6638429d7c).